report/report.py

```diff
@@ -540,15 +779,20 @@ def handler(event=None, context=None):
 
 if __name__ == "__main__":
 
+    t1_start = perf_counter()
+
     load_dotenv()
 
     connection = get_db_connection()
 
-    yesterday_data = load_yesterday_data(connection)
-
-    html_report = generate_html_string(yesterday_data)
+    html_report = generate_html_string(connection)
 
     pdf_file_path = './Bandcamp-Daily-Report.pdf'
 
     convert_html_to_pdf(html_report, pdf_file_path)
-    send_email(connection, pdf_file_path)
+
+    t1_stop = perf_counter()
+    print("Elapsed time report 2 during the whole program in seconds:",
+          t1_stop-t1_start)
+
+    # send_email(connection, pdf_file_path)
```

Comment:
Is this meant to be commented out ?

Reply:
Ahhh...I had it commented out so I didn't send emails when we exceeded our limit 😂